ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -772,6 +769,34 @@ open class DrawerController: UIViewController {
         }
     }
 
+    private func updateAppearance() {
+        let drawerTokens = DrawerController.drawerTokens
+        view.backgroundColor = backgroundColor
+        resizingHandleView?.backgroundColor = resizingHandleViewBackgroundColor
+        // if DrawerController is shown in UIPopoverPresentationController then we want to show different darkElevated color
+        if !useCustomBackgroundColor {
+            if presentationController is UIPopoverPresentationController {
+                backgroundColor = drawerTokens.popoverContentBackground
+            } else if useNavigationBarBackgroundColor {
+                backgroundColor = drawerTokens.navigationBarBackground
+            } else {
+                if presentationDirection.isVertical {
+                    backgroundColor = drawerTokens.drawerVerticalContentBackground
+                } else {
+                    backgroundColor = drawerTokens.drawerHorizontalContentBackground
+                }
+            }
+        }
+        guard let presentationController = (presentationController as? DrawerPresentationController) else {
+            return
+        }
+        presentationController.updateApperance()
+    }
+
+    var window: UIWindow? {
```

Comment:
My understanding is that this a requirement for `FluentUIWindowProvider` delegate. Since the window property is referenced for a controller, it's returning the value for its root view. Is there any other view that needs to conform to the delegate?

Reply:
Discussed this with Rafael offline. windowProvider need custom rootView to make assignment work. Resolving this for now